OutOfSchool/OutOfSchool.Common/Extensions/StringExtensions.cs

```diff
@@ -14,5 +14,10 @@ public static TEnum ToEnum<TEnum>(this string value, TEnum defaultValue)
 
             return Enum.TryParse<TEnum>(value, true, out var result) ? result : defaultValue;
         }
+
+        public static string Limit(this string value, int maxLength)
```

Comment:
Just a comment. Looks a little bit strange when you use `string.Limit(-2)` then it returns value. Maybe should think about returning an empty string if <= 0